packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -290,18 +290,38 @@ export const createBundle = async ({
                 } = await getProjectPkg()
                 const extendsTemplate = 'node_modules/' + ccExtensibility.extends
 
+                const pwaKitDeps = [
+                    '@salesforce/pwa-kit-dev',
+                    '@salesforce/pwa-kit-runtime',
+                    '@salesforce/pwa-kit-react-sdk'
+                ]
+
                 let cc_overrides: string[] = []
-                if (ccExtensibility.overridesDir) {
+                const packageVersions: {[key: string]: string} = {}
+
+                if (ccExtensibility.extends) {
                     const overrides_files = await walkDir(
                         ccExtensibility.overridesDir,
                         ccExtensibility.overridesDir
                     )
                     cc_overrides = Array.from(overrides_files).filter((item) =>
                         existsSync(path.join(extendsTemplate, item))
                     )
+
+                    // If the project is using template extensbility, we must look into the base template's package.json to find the versions of pwa-kit packages
+                    const extendsPkgJsonPath = path.join(extendsTemplate, 'package.json')
+                    if (existsSync(extendsPkgJsonPath)) {
+                        const {dependencies = {}, devDependencies = {}} = JSON.parse(
+                            readFileSync(extendsPkgJsonPath, 'utf-8')
+                        )
```

Comment:
```suggestion
                        const {dependencies = {}, devDependencies = {}} = await readJson(extendsPkgJsonPath)
```
`fs-extra` provides this as a util 😉